src/lib/controls/Input.svelte

```diff
@@ -49,13 +51,13 @@
 
 	function handleValidity(value: any, showErrors: boolean = true): boolean {
 		const validity = validate(value);
-		if (required) {
-			input.setCustomValidity(typeof validity === 'string' ? validity : validity ? '' : 'Invalid value.');
-			input.reportValidity();
+		if (required || validity !== null) {
+			input.setCustomValidity(typeof validity === 'string' ? validity : validity ? '' : 'Invalid value');
```

Comment:
Breaks some validated inputs, like "Variant of" and "Mission Pack" and many other inputs on mission edit page (allows submission of form when invalid data is present in the box but silently replaces the input with the last valid input)